_posts/research/2020-04-21-cooperative_multirobot_systems.md

```diff
@@ -0,0 +1,83 @@
+---
+title: Autonomous Coordination of Cooperative Multi-Robot Systems
+description: Reinforcement Learning for Distributed Control of Multiple Agents
+author: Joewie J. Koh
+permalink: research/cooperative_multirobot_systems.html
+image:
+    feature: research/multirobot/cooperative_manipulation.png
+    size: 75%
+excerpt_separator: <!-- More -->
+---
+
+Robotic systems with heterogeneous teams of cooperating robots have the potential to fundamentally change the paradigm of robotization. 
+These multi-robot systems hold the promise of drastically expanding industrial robotic applications: by coordinating multiple agents to cooperatively accomplish common goals, a system can be imbued with capabilities that are more than the sum of its parts.
+For example, while a team of robots might typically work independently, the occasional object that is too unwieldy or heavy for a single robot to manage could be cooperatively transported by a pair of robots.
+The emphasis on modularity allows for improved efficiency, robustness, and flexibility, reducing the capital requirement for state-of-the-art robotic systems and enabling more accessible automation across all segments of industry.
```

Comment:
1. You haven't dealt with or mentioned modularity before---it comes out of nowhere. Please talk about it, and also heterogeneity of robots that do different things.
2. Again, we don't focus on industrial applications--this is a more generic method and you want to be more theoretical rather than narrow it down to a specific use case. If you want to make specific examples, you should stay at an hypothetical level (and maybe provide two or three to not corner yourself somewhere you don't want to be).
3. Do you really want to talk about capital requirements? (this is a rhetorical question)